### PR TITLE
Add success variant to Tidbit

### PR DESCRIPTION
**Pull request by joe-watkins**
To support caution and error variants we are adding a success variant

### CODE REVIEW

**Review by tritland — DISMISSED** [review]

File: _checklist-web/tidbit.md [hunk 1]
```diff
@@ -114,8 +114,21 @@ Typical Tidbit markup consists of an SVG icon, heading, paragraph text, and a li
...
+
+{::nomarkdown}
+<example>
+{% include /examples/tidbit-success.html %}
+</example>
+{:/}
+
 ### Developer notes
-- Ensure the SVG icon has a meaningful text alternative. e.g. <code>aria-label="Error"</code> or <code>aria-label="Caution"</code>
+- Ensure the SVG icon has a meaningful text alternative. e.g. <code>aria-label="Error"</code>, <code>aria-label="Caution"</code>, <code>aria-label="Success"</code>
```
Comment (silverli) [inline _checklist-web/tidbit.md:130]: might want to update Line 19 as well with this kind of "success" example - but it's not critical

File: _includes/examples/tidbit-success.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="tidbit success">
+    <div class="icon">
+        <svg role="img" aria-label="Success" height="1.5em" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M64 32C28.7 32 0 60.7 0 96V416c0 35.3 28.7 64 64 64H384c35.3 0 64-28.7 64-64V96c0-35.3-28.7-64-64-64H64zM337 209L209 337c-9.4 9.4-24.6 9.4-33.9 0l-64-64c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0l47 47L303 175c9.4-9.4 24.6-9.4 33.9 0s9.4 24.6 0 33.9z"/></svg>
+    </div>
+    <div class="content">
+        <h2 id="tidbit-heading-error">That went well!</h2>
```
Comment (silverli) [inline _includes/examples/tidbit-success.html:6]: I'd change the id to "tidbit-heading-success"

same with aria-describedby reference below

File: _sass/base/_colors.scss [hunk 3]
```diff
@@ -44,4 +44,6 @@ $tdds-critical-300: #D20D00;
 $tdds-critical-100: #FFEDEC;
 $tdds-caution-200: #F8A50C;
 $tdds-caution-200-darkened: #CC8400;
-$tdds-caution-100: #FFF3DE;
\ No newline at end of file
+$tdds-caution-100: #FFF3DE;
+$tdds-success-100: #E6FCE8;
```
Comment (silverli) [inline _sass/base/_colors.scss:48]: I'm noticing the warning/caution icon color against the yellow background is not meeting 3:1. Maybe we can make the yellow background lighter or icon darker.

**Review by silverli — CHANGES_REQUESTED** [review]
Great work, just a couple o' thoughts!

**Comment by joe-watkins** [comment]
@silverli thanks for the fixes! Good catches. I've addressed them all.

**Review by silverli — APPROVED** [review]
Looks great!

**Pull request merged**